pkg/dnservice/store_rpc_handler.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/pb/metadata"
```

Comment:
You can not add tracing here.  You should add trace in the txn/rpc/sender and server.go